lib/Detabinator.js

```diff
@@ -1,20 +1,3 @@
-/**
- *
- * Copyright 2016 Google Inc. All rights reserved.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
```

Comment:
This has to stay. This is not Our license....

Reply:
add external licenses here https://github.com/demos-europe/demosplan-js-utils/pull/6/commits/287c0eca60ddb128a4ec189be8655515b1b9eb54

Reply:
These two files are in the LICENSE file. Is it not enough?

Reply:
No, it is not. The license needs to be attributed in the file. Moreover, the license requires the section in the LICENSE file to be included